docs/users/Instrument_scientist.rst

```diff
@@ -49,4 +50,4 @@ It will open a page like below
 One can then submit several runs for re-reduction.
 This is useful if there is a mistake in the reduction script.
 The messages for postprocessing are not necessarily handled in the order of run numbers.
-Most instrument scientists do not have this option.
+**Most instrument scientists do not have this option.**
```

Comment:
Would it be better to do this as a [`.. note::`](https://pythonhosted.org/an_example_pypi_project/sphinx.html#paragraph-markup)?